website/project/model.py

```diff
@@ -2173,6 +2186,7 @@ def fork_node(self, auth, title='Fork of '):
         # Clear permissions before adding users
         forked.permissions = {}
         forked.visible_contributor_ids = []
+        forked.save()
```

Comment:
Importing `NotificationSubscription` into `website.project.model` would cause a circular import? Do I understand you right?